development/cli/getting_started.rst

```diff
@@ -0,0 +1,67 @@
+===============
+Getting started
+===============
+
+The command line interface (CLI) is a useful utility for phpBB administrators who have access to a shell on the server they run phpBB on, as well as for extension developers. It provides CLI commands for managing config values, extensions, running database migration, purging the cache, and more.
+
+Most operating systems ship with a built in command line application. For Mac OS and Ubuntu it is called "Terminal" and for Windows it is called "Command Prompt". Third party software such as PuTTy and iTerm can also be used.
```

Comment:
Download and install Putty basically. Any further details they can get elsewhere tbh (type in the ip, username, port in the relevant tabs and hit connect)